app/views/add-reduction.html

```diff
@@ -0,0 +1,164 @@
+{% extends "includes/layout.html" %} 
+
+{% block content %}
+
+<main id="content" role="main">
+  {% include "includes/breadcrumbs.html" %} 
+  
+  <header class="heading-xlarge" style="margin-top:0; margin-bottom:0">
+    New reduction
+  </header>
+
+  {%if failureText %}
+  <p class="message message--failure">{{ failureText }}</p>
+  {% endif %}
+
+  <p class="lede">Add a new reduction for <span class="bold">{{ title }}</span></p>
+  {% set first = true %}
+  <form action="/offender-manager/{{ linkId }}/add-reduction" method=POST id="reductionForm">
+    <div class="form-group" id="reduction_reason_selection">
+      <label class="form-label-bold" for="select-box">Reason for reduction</label>
+      <select class="form-control" id="select-box" name="reasonForReductionId" onChange="populateHours(), calculateExpiry()">        
+        <option value="" disabled selected>Please select a reason...</option>
+        {% for reason in referenceData %}
+        <option value="{{ reason.id }}">{{ reason.reasonShortName }}</option>
+        {% else %}
```

Comment:
Question: What does this else do?

Reply:
I don't know....and I don't know why that didn't break..... 😕 
Removed now anyway